Tests/js/views/assets/ez-dashboardblocksview-tests.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblocksview-tests', function (Y) {
+    var renderTest,
+        forwardActiveTest,
+        addBlockTest,
+        removeBlockTest,
+        VIEW_CONFIG = {container: '.container'},
+        SELECTOR_CONTENT = '.dashboard-content',
+        View = Y.eZ.DashboardBlocksView;
+
+    Y.eZ.DashboardBlockBaseView = Y.Base.create('dashboardBlockBaseView', Y.View, [], {}, {
+        ATTRS: {
+           priority: {},
+            identifier: {}
+        }
+    });
+
+    renderTest = new Y.Test.Case({
+        name: 'eZ Dashboard Blocks View render test',
+
+        setUp: function () {
+            this.view = new View(VIEW_CONFIG);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        'Should render view correctly without blocks inside': function () {
+            var templateCalled = false,
+                origTpl = this.view.template;
+
+            this.view.template = function () {
+                templateCalled = true;
+
+                return origTpl.apply(this, arguments);
+            };
+
+            this.view.render();
+
+            Y.Assert.isTrue(templateCalled, 'The template should have been used to render view');
+        }
+    });
+
+    addBlockTest = new Y.Test.Case({
+        name: 'eZ Dashboard Blocks View add block tests',
```

Comment:
the tests in this test case are wrongly written. You are testing `addBlock` not `render`, so you should basically get 3 tests here:
* `Should add the block to the list` where you add a block and assert it is in the `blocks` attribute, its bubble target was set to the dashboard view and the active flag value was on the block.
* `Should remove an existing block with the same identifier` to check the case where you add a block with an existing identifier (I would to not have some code for that, but you insist on having, so you test to it explicitly)
* `Should keep the block list sorted` where you add several block with different priorities to make sure the sorting is done